integration-test/mqtt/mqtt_system_test.c

```diff
@@ -574,12 +574,9 @@ static void eventCallback( MQTTContext_t * pContext,
     TEST_ASSERT_EQUAL( MQTTSuccess, pDeserializedInfo->deserializationResult );
     pPublishInfo = pDeserializedInfo->pPublishInfo;
 
-    if( ( pPacketInfo->type == disconnectOnPacketType ) ||
-        ( ( pPacketInfo->type & 0xF0U ) == disconnectOnPacketType ) )
+    if( ( pPacketInfo->type == packetTypeForDisconnection ) ||
+        ( ( pPacketInfo->type & 0xF0U ) == packetTypeForDisconnection ) )
     {
-        /* Terminate MQTT connection with server for session restoration test. */
-        TEST_ASSERT_EQUAL( MQTTSuccess, MQTT_Disconnect( &context ) );
```

Comment:
Was this moved?

Reply:
This was removed as the `MQTT_Disconnect` call is not explicitly made to terminate the MQTT connection, but the network connection is severed directly by calling `OpenSsl_Disconnect`.

Only terminating the network connection is required for terminating an MQTT session